scripts/travis-yarn.sh

```diff
@@ -7,10 +7,6 @@ then
 
   echo "Starting server..."
   yarn run serve &
```

Comment:
I would run the script here but rename it "wait-for-server.sh" and then just make it exit when the server is ready.
```
bash scripts/wait-for-server.sh || exit 1
```